dev-docs/modules/userId.md

```diff
@@ -495,10 +580,13 @@ Bidders that want to support the User ID module in Prebid.js, need to update the
 {: .table .table-bordered .table-striped }
 | ID System Name | ID System Host | Prebid.js Attr | Example Value |
 | --- | --- | --- | --- | --- | --- |
-| PubCommon ID | n/a | bidRequest.userId.pubcid | `"1111"` |
-| Unified ID | Trade Desk | bidRequest.userId.tdid | `"2222"` |
+| CriteoRTUS | Criteo | bidRequest.userId.criteortus | `"1111"` |
 | DigiTrust | IAB | bidRequest.userId.digitrustid | `{data: {id: "DTID", keyv: 4, privacy: {optout: false}, producer: "ABC", version: 2}` |
-| ID5 ID | ID5 | bidRequest.userId.id5id | `"ID5-12345"` |
+| ID5 ID | ID5 | bidRequest.userId.id5id | `"1111"` |
+| IdentityLink | Trade Desk | bidRequest.userId.idl_env | `"1111"` |
+| LiveIntent ID | Live Intent | bidRequest.userId.liveIntentId | `"1111"` |
```

Comment:
Hello @jankoulaga , @bretg 
I see that in bidRequest  object LiveIntent Id is passed as , ``` bidUserId.lipb.lipbid ```
Refer: https://github.com/prebid/Prebid.js/pull/4178/files#diff-93e5cbcbb2aa2045b6e254010a1fc6b4R729
But in docs here, it is mentioned as ``` bidRequest.userId.liveIntentId ```
Can you please check and let us know?

Reply:
You're right @pm-harshad-mane. The path should be correct now.

Reply:
Thank you @jankoulaga !!